src/lib/bootloader/sections.rb

```diff
@@ -44,7 +42,31 @@ def default=(value)
     # writes default to system making it persistent
     def write
       return if default.empty?
-      Yast::Execute.on_target("/usr/sbin/grub2-set-default", default)
+
+      Yast::Execute.on_target("/usr/sbin/grub2-set-default", title_to_path(default))
+    end
+
+  private
+
+    def read_default
```

Comment:
I still think you should document methods no matter whether they are private or not.
```
# @return path of the default entry
# @return title of the default entry
```